volatility/framework/plugins/windows/memmap.py

```diff
@@ -43,22 +47,40 @@ def _generator(self, procs):
                     pid, excp.invalid_address, excp.layer_name))
                 continue
 
+            filename = str(pid)
```

Comment:
Why does this need str()?

Reply:
Pid's an int, not a string, and we're going to cat a string on the end of it, so it needs to be cast...

Reply:
I see...I was confused because I think format() will convert it to a string anyway, but explicitly doing it before passing to format() works too.

Reply:
Oh, no, you're right.  I guess I was pulling it out for debugging, we might as well pass it straight to the format thing instead of putting it in an holding variable for one line...  5:)